lib/connection.js

```diff
@@ -139,6 +139,8 @@ Connection.prototype.open = function (callback) {
       self.bindSocketListeners();
       self.startup(callback);
     });
+    // TLSSocket will validate for values from 512 to 16K (depending on the SSL protocol version)
+    this.netClient.setMaxSendFragment(this.options.socketOptions.coalescingThreshold);
```

Comment:
It looks like in the case of coalescingThreshold > 16K, the setting just isn't applied?   So this would not be useful unless the coalescingThreshold < 16k.

Reply:
correct